buildSrc/src/main/java/io/opentelemetry/android/PublishPlugin.java

```diff
@@ -0,0 +1,112 @@
+package io.opentelemetry.android;
+
+import com.android.build.api.dsl.LibraryExtension;
+
+import org.gradle.api.Plugin;
+import org.gradle.api.Project;
+import org.gradle.api.plugins.PluginContainer;
+import org.gradle.api.publish.PublishingExtension;
+import org.gradle.api.publish.maven.MavenPublication;
+import org.gradle.plugins.signing.SigningExtension;
+
+import java.util.Locale;
+
+import kotlin.Unit;
+
+/**
+ * This plugin takes care of configuring a library project by creating a maven publication that
+ * contains the library artifact and (when the "-Prelease=true" param is present in the gradle command) it will also include
+ * the library's javadoc, sources artifacts and all the artifacts will be signed.
+ */
+public class PublishPlugin implements Plugin<Project> {
```

Comment:
How would you feel about sticking with kotlin for the gradle/build tooling? Might be a little unfortunate to have both java and kotlin hanging around in `buildSrc`.

Reply:
I don't have a strong opinion about it so I'm fine with using kotlin to keep consistency 👍 I'll add the changes.